app/views/results/student/_student_panes.html.erb

```diff
@@ -5,6 +5,9 @@
         <ul>
           <li><a href='#code_holder'><%= t('marker.source_code') %></a></li>
           <li><a href='#annotations_summary'><%= t('marker.annot_summary') %></a></li>
+          <% if has_test_results?(submission) %>
```

Comment:
@WilsonChiang Can you please add a check to see if `assignment.enable_test` is true? (Both here and in the marker_panes.)